### PR TITLE
Master

### PR DESCRIPTION
**Pull request by zauker**
Add example for use addRoute function.

### CODE REVIEW

**Comment by zauker** [comment]
There are two different between this function and the original one of i18next-node:

1) ensureLngSupport
It was not included the ensureIsSupportedLanguage function

2) parameter: lng
this is converted with the value of this language to avoid that you can have the incorrect routes.
example:
if I have these routes:
/en/team
/fr/equipe
/it/squadra

if you leave the parameter: lng as a parameter would also agree to the following routes:
/it/team
/fr/team
/en/equipe
/it/equipe
/en/squadra
/fr/squadra

these routes are not correct and should not be accessible, for this reason I prefer put the language value into :lng parameter for localized routes

File: README.md [hunk 1]
```diff
@@ -53,6 +53,56 @@ app.post('/locales/add/:lng/:ns', middleware.missingKeyHandler(i18next));
...
+    i18nextMiddleware.addRoute(i18next, '/:lng/key-to-translate', ['en', 'de', 'it'], app, 'get', function(req, res) {
+      //endpoint function
+    });
+  });
+app.use(i18nextMiddleware.handle(i18next));
+app.listen(port, function(){ console.log('Server listening on port', port) } );
+```
+
+```js
+var express = require('express'),
```
Comment (jamuhl) [inline README.md]: isn't this the same as on line 61?

**Comment by jamuhl** [comment]
great...will be merged and published with next version....might get to this later the day...

thanks a lot for making this better.

**Comment by jamuhl** [comment]
in published v0.4.0

**Pull request merged**